src/main/kotlin/org/rust/lang/core/types/infer/TypeInference.kt

```diff
@@ -126,20 +129,35 @@ class RsInferenceContext(
         }
     }
 
-    fun infer(fn: RsFunction): RsInferenceResult {
-        extractParameterBindings(fn)
+    fun infer(fn: RsTypeInferenceRoot): RsInferenceResult {
```

Comment:
`fn` does not look like the best name anymore :)

Reply:
Same issue about name of `RsFnInferenceContext`